dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -3577,31 +2879,31 @@
         "documents"
       ]
     },
-    "AbstractiveSummaryDocumentResultWithDetectedLanguage": {
+    "AbstractiveSummarizationResult": {
       "type": "object",
-      "description": "An object representing the summarization result of a single document with detected language.",
+      "description": "An object representing the pre-build summarization results of each document.",
       "allOf": [
         {
-          "$ref": "#/definitions/AbstractiveSummaryDocumentResult"
+          "$ref": "#/definitions/AbstractiveSummarizationResultBase"
         },
         {
-          "$ref": "#/definitions/DocumentDetectedLanguage"
+          "$ref": "common.json#/definitions/PreBuiltResult"
         }
+      ],
+      "required": [
+        "documents"
       ]
     },
-    "AbstractiveSummarizationResult": {
+    "AbstractiveSummaryDocumentResultWithDetectedLanguage": {
```

Comment:
I thought auto language detection would not GA, should this type have the detected language property removed?